src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/MicrosoftAzureSqlServerDataSourceItemFixture.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Reveal.Sdk.Dom.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class MicrosoftAzureSqlServerDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_SetsTitleAndDatasource_AsProvided()
+        {
+            // Arrange
+            var dataSource = new MicrosoftAzureSqlServerDataSource();
+            var title = "Test title";
+
+            // Act
+            var dataSourceItem = new MicrosoftAzureSqlServerDataSourceItem(title, dataSource);
+
+            // Assert
+            Assert.Equal(title, dataSourceItem.Title);
```

Comment:
I think we can use Theory to increase the coverage of unit tests. For ex the commit: [https://github.com/RevealBi/Reveal.Sdk.Dom/pull/93/commits/051374207ac976bf40077db08ac82c5c5948b088](url)

The base constructor should also check: 
- Assert.Equal(dataSource.Id, dataSourceItem.DataSourceId);
- Assert.Equal(expectedDSTitle, dataSourceItem.DataSource.Item);